int8    plan_mode
int8    predict_mode
int32[] moving_humans
int32[] still_humans
